src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -577,6 +577,7 @@
     <Reference Include="System.Data" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="Microsoft\Data\SqlClient\SqlClientEventSource.cs" />
```

Comment:
I would recommend not making any changes to netcore folder and keeping it primarily for NetFx in this PR. Since our goal is to re-use the file by sharing it between both projects when working on NetCore, it doesn't make sense to commit 2 files first and remove 1 later. This is a no-op anyway.